<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowInfoWidget</class>
 <widget class="QWidget" name="BorrowInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>725</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>操作</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="4">
         <widget class="QCheckBox" name="chk_relend">
          <property name="text">
           <string>可续借</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="chk_over">
          <property name="text">
           <string>逾期记录</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QCheckBox" name="chk_all">
          <property name="text">
           <string>查看全部</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="btn_reLend">
          <property name="text">
           <string>续借</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QCheckBox" name="chk_noreturn">
          <property name="text">
           <string>只看未还</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>图书编号</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="placeholderText">
           <string>读者编号</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBorrow">
       <property name="title">
        <string>@_@</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_returnBook">
          <property name="text">
           <string>还书</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
